storage/simulation/cluster.go

```diff
@@ -64,6 +69,8 @@ func createCluster(stopper *stop.Stopper, nodeCount int) *Cluster {
 		nodes:         make(map[proto.NodeID]*Node),
 		stores:        make(map[proto.StoreID]*Store),
 		ranges:        make(map[proto.RangeID]*Range),
+		rand:          rand,
+		seed:          seed,
```

Comment:
Are you expecting that simulations will be deterministic for a given seed? Or is there some other reason for storing the seed here?